src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -21,6 +22,15 @@
     <None Remove="UI\Images\Alignment\align_x_distribute.png" />
     <None Remove="UI\Images\Alignment\align_y_average.png" />
     <None Remove="UI\Images\Alignment\align_y_distribute.png" />
+    <None Remove="UI\Images\close_default_16px_blue.png" />
```

Comment:
We are already using a blue close button image for the dialogs
https://github.com/DynamoDS/Dynamo/blob/67489bf9a2ccd08317e285493a5171dfd8ef9785/src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml#L2825? Why are we using different ones for these dialogs?

Reply:
Oh I see you are replacing it with new images. In that case, remove the old ones along with references in the project file.

Reply:
Should I? I wasn't sure if they are still in use and wanted to give it a moment and see if anyone would raise a flag. I will remove the old icon versions then.

Reply:
Is any control using them after these changes?